setup.py

```diff
@@ -6,7 +6,7 @@
 
 REPO_ROOT = path.abspath(path.dirname(__file__))
 
-VERSION = "4.3.0"
+VERSION = "4.4.0"
```

Comment:
```suggestion
VERSION = "4.3.0"
```

A bit easier if we leave version bumps to their own PR. 😄

Reply:
Okay I removed the version update commit.